src/components/MoneyRequestConfirmationList.js

```diff
@@ -206,7 +206,6 @@ const defaultProps = {
     shouldShowSmartScanFields: true,
     isPolicyExpenseChat: false,
     iou: iouDefaultProps,
-    policyTaxRates: {},
```

Comment:
Do we need a default empty object for policy?

Reply:
I think this File isn't even in use. But I can update it. Thanks.